api_types.ts

```diff
@@ -1047,6 +1070,14 @@ export interface SyncExecutionContext<
    * Information about state of the current sync.
    */
   readonly sync: Sync<ContinuationT, IncrementalContinuationT, IncrementalSyncContinuationT>;
+
+  /**
+   * A service for retrieving the sync state in Coda Brain.
+   * @hidden
+   *
+   * TODO(ebo): unhide this
+   */
+  readonly syncState: SyncStateService;
```

Comment:
Can we maybe call this `syncStateService`? Isn't `sync` an object that already represents "sync state"?

Reply:
I guess it might be clearer if we include `service` in the name? wdyt @erickoledadevrel

Reply:
I like adding "Service"